source/Command.h

```diff
@@ -135,14 +142,18 @@ class Command {
 
 
 private:
-	explicit Command(uint32_t state);
-	Command(uint32_t state, const std::string &text);
+	explicit Command(const CommandState &state);
+	Command(const CommandState &state, const std::string &text);
+
+	// Returns a CommandState with only one bit set. This is used for its
+	// static assertion in Command.cpp, and is available nowhere else.
+	template<unsigned bit>
+	static CommandState SetBit();
 
 
 private:
-	// The key commands are stored in a single bitmask with
-	// 32 bits for key commands.
-	uint32_t state = 0;
+	// The key commands are stored in a CommandState.
+	CommandState state = 0;
```

Comment:
```suggestion
	CommandState state;
```

default constructor initializes it to 0 already.

Reply:
the `= 0` is gone